code/game/gamemodes/cult/talisman.dm

```diff
@@ -231,7 +231,7 @@
 	. = ..()
 	user.visible_message("<span class='warning'>[user]'s hand flashes a bright blue!</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, emitting an EMP blast.</span>")
-	empulse(src, 4, 8, 1)
+	empulse(src, 4, 8, 1,, TRUE)
```

Comment:
What happened to the double ,, here?

Reply:
`/proc/empulse(turf/epicenter, heavy_range, light_range, log=0, cause = null, srccult = FALSE)` empulse calls for 5 arguments normally, I didnt want to supply information in the wrong slot.. if you understand what I mean. I want to pass in TRUE for the 6th argument, not the 5th. So I added another comma to make TRUE pass into the 6th.

Reply:
/proc/empulse(turf/epicenter, heavy_range, light_range, log=0, cause = null, srccult = FALSE) empulse calls for 5 arguments normally, I didnt want to supply information in the wrong slot.. if you understand what I mean. I want to pass in TRUE for the 6th argument, not the 5th. So I added another comma to make TRUE pass into the 6th.